docs/guide/advanced/sfc.md

```diff
@@ -295,6 +301,90 @@ The `i18n` custom blocks below of `json5` format:
 </i18n>
 ```
 
+### Define global format
+
+If you are using one of `@intlify/vite-plugin-vue-i18n`, `@intlify/vue-i18n-loader`  or `@intlify/rollup-plugin-vue-i18n`
```

Comment:
Unfortunately, we can use `globalSFCScope` and `defaultSFCLocale` options, so `@intlify/rollup-plugin-vue-i18n` only.
We need to mention at docs with `:::warning:::`